domain-model-assistant/Assets/Components/Scripts/lineCollider.cs

```diff
@@ -78,14 +82,15 @@ void OnMouseOver()
     private List<Vector2> CalculateColliderPoints() 
     {
         //Get All positions on the line renderer
-        Vector3[] positions = edge.GetPositions();
+        Vector3 linePos1 = edge.GetPosition1();
+        Vector3 linePos2 = edge.GetPosition2();
 
         //Get the Width of the Line
         // float width = 0.5f;
         float width = 2f * edge.GetWidth();
 
         //m = (y2 - y1) / (x2 - x1)
-        float m = (positions[1].y - positions[0].y) / (positions[1].x - positions[0].x);
+        float m = (linePos2.y - linePos1.y) / (linePos2.x - linePos1.x);
```

Comment:
Rename `m` to `slope` to make the variable name clearer. Also, are you handling the case where `linePos2.x - linePos1.x == 0`?